lib/pinga-server/src/server.rs

```diff
@@ -361,17 +366,26 @@ async fn receive_job_requests(
     Ok(())
 }
 
+const CONCURRENT_TASKS: AtomicUsize = AtomicUsize::new(0);
+
```

Comment:
same thing here! Something like 
``` #[instrument (level="info", skip(rx)]```

Reply:
Why would we want a span for the task that spawns task?

Reply:
Something like this?

![image](https://github.com/systeminit/si/assets/6289779/935e1fa7-4907-4632-9237-16a2d7b9c421)

Reply:
yep!!

Reply:
Done!